src/simulator_core/solver/network/assets/fall_type.py

```diff
@@ -96,6 +96,13 @@ def get_equations(self) -> list[EquationObject]:
             A list of EquationObjects that contain the indices, coefficients, and right-hand side
             values of the equations.
         """
+        # Check if there are two nodes connected to the asset
+        if len(self.connected_nodes) != 2:
+            raise ValueError("The number of connected nodes must be 2!")
+        # Check if the number of unknowns is 6
+        if self.number_of_unknowns != 6:
```

Comment:
Deze lijkt een beetje raar, want je zet hem erboven al op die waarde, waarom checken we dit nog hier?

Reply:
Number of unknowns is niet verborgen, je kan het dus overrulen...

Reply:
Als we dit willen aanpassen moeten we even een issue maken.